src/frontend/ts/types/urbitChess.ts

```diff
@@ -32,7 +32,9 @@ export enum Update {
   Position = 'position',
   Result = 'result',
   DrawOffer = 'draw-offer',
-  DrawDeclined = 'draw-declined'
+  DrawDeclined = 'draw-declined',
+  DrawClaimAvailable = 'draw-claim-available',
+  SpecialDrawDeclined = 'special-draw-declined'
```

Comment:
Don't need `SpecialDrawDeclined` - making a move instead of taking the draw is an implicit decline